phpstan.neon

```diff
@@ -1,38 +1,49 @@
+parametersSchema:
+  # Copied from https://github.com/ekino/phpstan-banned-code/blob/v0.4.0/extension.neon.
+  banned_code: structure([
+    nodes: listOf(structure([
+      type: string()
+      functions: schema(listOf(string()), nullable())
+    ]))
+  ])
+
 parameters:
+  # Disable all phpstan rules.
+  customRulesetUsed: true
   excludePaths:
     - /app/vendor/*
     - */assets/src/lib/campaignmonitor/samples/*
     - */tests/*
     - */bootstrap/scripts/*
   scanDirectories:
     - /app
-  level: 0
   fileExtensions:
     - php
     - theme
     - inc
   reportUnmatchedIgnoredErrors: false
   ignoreErrors:
-    - "#Function (.*) not found#"
```

Comment:
This looks fine to me, but we might want to trial it on a few canaries just to see how this impacts some real use cases.